src/main/java/seedu/eylah/expensesplitter/logic/parser/ParserUtil.java

```diff
@@ -76,7 +78,9 @@ public static ItemPrice parseItemPrice(String priceInString) throws ParseExcepti
     public static Amount parseAmount(ItemPrice price, int numPersons) throws ParseException {
         requireAllNonNull(price, numPersons);
         BigDecimal bigDecimalNumPersons = new BigDecimal(numPersons);
-        BigDecimal pricePerPerson = price.getItemPrice().divide(bigDecimalNumPersons);
+        BigDecimal itemPrice = price.getItemPrice();
+        BigDecimal pricePerPerson = itemPrice.divide(bigDecimalNumPersons,
+                new MathContext(4, RoundingMode.HALF_UP));
```

Comment:
Maybe you could include some comment on this part to explain the reason using `MathContext` to do the rounding.